File: Modul-1/Soal 8/PRAK108-2410817210014-ZAVIER GHANI.py
PI = 3.14159265358979323846
def cetak_pesan():
    putaran = 5
    jarak_tempuh = 14.0  # dalam kilometer
    keliling_per_putaran = jarak_tempuh / putaran
    jari_jari = keliling_per_putaran / (2 * PI)

    print("Diketahui :")
    print(f"Pak Dengklek mengelilingi taman = {putaran} Putaran")
    print(f"Jarak tempuh Pak Dengklek = {jarak_tempuh} Kilometer")
    print("Jawaban :")
    print(f"Jari-jari taman yang dikelilingi Pak Dengklek adalah {jari_jari:.2f} Kilometer")

cetak_pesan()

File: Modul-1/Soal 8/PRAK108-2410817210014-ZAVIERGHANI..c
#include <stdio.h>
#define PI 3.14159265358979323846

int main() {
    int putaran = 5;
    float jarak_tempuh = 14.0; 

    float keliling_per_putaran = jarak_tempuh / putaran;

    float jari_jari = keliling_per_putaran / (2 * PI);

    printf("Diketahui :\n");
    printf("Pak Dengklek mengelilingi taman = %d Putaran\n", putaran);
    printf("Jarak tempuh Pak Dengklek = %.0f Kilometer\n", jarak_tempuh);
    printf("Jawaban :\n");
    printf("Jari-jari taman yang dikelilingi Pak Dengklek adalah %.2f Kilometer\n", jari_jari);

    return 0;
}
